TWReader/UIBroker.cls.xml

```diff
@@ -29,7 +29,7 @@ fields must be appropriate js-array</Description>
 	
 	do ..SaveAllPosts(shouldCountEC)
 	if ($length(fields) < 1) {
-		set fields = "['sourceId','title', 'date', 'time', 'city', 'id', 'EC']"
+		set fields = "['query', 'title', 'date', 'time', 'city', 'id', 'EC' , 'latitude', 'longitude']"
```

Comment:
@ssolomaha after refactoring I'll introduce in branch `map-colorer` we'll be able to just call DrawTable with other set of arguments!